Panics on fresh repository with no tags yet

Heyho, i'm workin on a new lib, that i just started 10 mins ago. 
Bumptag does not seem to like being run on a new repository, when there are no tags yet. 

```sh
tobias@tobias-P870DM-G:~/go/src/github.com/Nerzal/atreugo-swagger$ bumptag
panic: command 'git describe --tags --abbrev=0' failed: exit status 128
fatal: Keine Namen gefunden, kann nichts beschreiben.

goroutine 1 [running]:
main.panicIfError(...)
        /home/tobias/go/pkg/mod/github.com/sv-tools/bumptag@v1.5.0/bumptag.go:298
main.main()
        /home/tobias/go/pkg/mod/github.com/sv-tools/bumptag@v1.5.0/bumptag.go:370 +0x9d3
```

> fatal: Keine Namen gefunden, kann nichts beschreiben.

more or less translates to 
> fatal: No names found, cannot describe

--- a/bumptag.go
+++ b/bumptag.go
@@ -114,15 +114,16 @@ func gitConfigBool(name string, defaultValue bool) bool {
 }
 
 func findTag() (*semver.Version, string, error) {
-	output, err := git("", "describe", "--tags", "--abbrev=0")
+	output, err := git("", "tag")
 	if err != nil {
-		if strings.Contains(err.Error(), "No names found") {
-			return &semver.Version{}, "", nil
-		}
 		return nil, "", err
 	}
 	if output == "" {
 		return &semver.Version{}, "", nil
+	}
+	output, err = git("", "describe", "--tags", "--abbrev=0")
+	if err != nil {
+		return nil, "", err
 	}
 	currentTagName := output
 	if !strings.HasPrefix(output, tagPrefix) {

--- a/bumptag_test.go
+++ b/bumptag_test.go
@@ -314,29 +314,49 @@ func TestFindTag(t *testing.T) {
 	defer tearDown()
 
 	ctrl.EXPECT().
-		Git("", "describe", "--tags", "--abbrev=0").
-		Return("", errors.New("test-error"))
+		Git("", "tag").Return("", errors.New("test-error"))
 	_, _, err := findTag()
-	assert.Error(t, err)
+	assert.Error(t, err, "test-error")
 
 	ctrl.EXPECT().
-		Git("", "describe", "--tags", "--abbrev=0").
-		Return("", nil)
+		Git("", "tag").Return("", nil)
 	tag, tagName, err := findTag()
 	assert.NoError(t, err)
 	assert.Equal(t, "", tagName)
 	assert.Equal(t, "0.0.0", tag.String())
 
+	tagCall := ctrl.EXPECT().
+		Git("", "tag").Return("text-tag", nil)
 	ctrl.EXPECT().
 		Git("", "describe", "--tags", "--abbrev=0").
-		Return("text-tag", nil)
+		Return("", errors.New("test-error")).After(tagCall)
+	_, _, err = findTag()
+	assert.Error(t, err, "test-error")
+
+	tagCall = ctrl.EXPECT().
+		Git("", "tag").Return("text-tag", nil)
+	ctrl.EXPECT().
+		Git("", "describe", "--tags", "--abbrev=0").
+		Return("1.2.3", nil).After(tagCall)
+	tag, tagName, err = findTag()
+	assert.NoError(t, err)
+	assert.Equal(t, "1.2.3", tagName)
+	assert.Equal(t, "1.2.3", tag.String())
+
+	tagCall = ctrl.EXPECT().
+		Git("", "tag").Return("text-tag", nil)
+	ctrl.EXPECT().
+		Git("", "describe", "--tags", "--abbrev=0").
+		Return("text-tag", nil).After(tagCall)
 	_, _, err = findTag()
 	assert.Error(t, err)
 
 	tagPrefix = "v"
+	tagCall = ctrl.EXPECT().
+		Git("", "tag").Return("text-tag", nil)
 	ctrl.EXPECT().
 		Git("", "describe", "--tags", "--abbrev=0").
-		Return("v-text-tag", nil)
+		Return("v-text-tag", nil).After(tagCall)
 	_, _, err = findTag()
 	assert.Error(t, err)
 }

